pkg/plugins/packages.go

```diff
@@ -0,0 +1,164 @@
+package plugins
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"strings"
+
+	"github.com/joho/godotenv"
+	"github.com/mudler/yip/pkg/logger"
+	"github.com/mudler/yip/pkg/schema"
+	"github.com/twpayne/go-vfs/v4"
+)
+
+type Installer string
```

Comment:
mmmmm just a design nit: would probably move this to an interface and the distro as implementations, but shouldn't be blocking

Reply:
I would actually move this to be constants or something instead no? I dunno, doesn't really matter to me, it's just a way of making it easier to understand the checks IMHO

Reply:
Ah crap, this is the installer. Sorry I thought it was the distro vars

Reply:
mmmh, after having a quick go, I do indeed have the opinion that it may make things less clear, so I rather not. I like it how it is now, very straightforward, very simple, easily extensible with a few lines to add a new installer if needed. 

wdyt?

Reply:
looks good ! was just a small nit from my pov, nothing really blocking, let's go!